client/src/commons/CircleSelect/CircleSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Select, SelectProps, FormControl } from '@material-ui/core';
+
+import useStyles from './CircleSelectStyles'
+
+const CircleSelect: React.FC<Props> = (props: Props): JSX.Element => {
+    const classes = useStyles({});
+    const {options, ...rest} = props;
+
+    return (
+        <FormControl variant='outlined' className={classes.formControl}>
+            <Select
+                native
+                {...rest}
+                className={classes.selectBorder}
+                classes={{ root: classes.unsetSelectColor }}
+            >
+                {
+                    options.map((option, index) => {
+                        return <option key={index} value={index}>{option}</option>
```

Comment:
change value to index

Reply:
change value={index} to value={option}

Reply:
since option is an object it shouldnt even use index,
we have 2 options -
1. assume every option has an id and name 
and use the id as value and key and the name as the displayed child of option 
2. just return props.children and let the user of the component decide